### PR TITLE
Update 2.0 README to include missing models.

### PR DESCRIPTION
**Pull request by emackey**

### CODE REVIEW

File: 2.0/README.md [hunk 1]
```diff
@@ -40,11 +45,10 @@
 | [Barramundi Fish](BarramundiFish)             | ![](BarramundiFish/screenshot/screenshot.jpg)    | :white_check_mark: | :white_check_mark: |                    |
 | [Boom Box](BoomBox)                           | ![](BoomBox/screenshot/screenshot.jpg)           | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 | [Corset](Corset)                              | ![](Corset/screenshot/screenshot.jpg)            | :white_check_mark: | :white_check_mark: |                    |
+| [Damaged Helmet](DamagedHelmet)               | ![](DamagedHelmet/screenshot/screenshot.png)     | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```
Comment (bghgary) [inline 2.0/README.md]: Can the screenshot be resized? It looks out of place.

File: 2.0/README.md [hunk 2]
```diff
@@ -40,11 +45,10 @@
...
 | [Flight Helmet](FlightHelmet)                 | ![](FlightHelmet/screenshot/screenshot.jpg)      | :white_check_mark: | :white_check_mark: |                    |
 | [Lantern](Lantern)                            | ![](Lantern/screenshot/screenshot.jpg)           | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 | [Water Bottle](WaterBottle)                   | ![](WaterBottle/screenshot/screenshot.jpg)       | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| [Metal Rough Spheres](MetalRoughSpheres)      | ![](MetalRoughSpheres/screenshot/screenshot.png) |                    |                    |                    |
-| [Normal Tangent Test](NormalTangentTest)      | ![](NormalTangentTest/screenshot/screenshot.png) | :white_check_mark: |                    |                    |
 | [Boom Box With Axes](BoomBoxWithAxes)         | ![](BoomBoxWithAxes/screenshot/screenshot.jpg)   | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```
Comment (bghgary) [inline 2.0/README.md]: I would argue this should also go in the feature tests section.

File: 2.0/README.md (hunk 1 above)
Comment (emackey) [reply 2.0/README.md]: I pushed a smaller one, but I don't see it yet on GitHub.  Could be a caching issue...

File: 2.0/README.md (hunk 2 above)
Comment (emackey) [reply 2.0/README.md]: Done.

**Pull request merged**